### PR TITLE
Frontend.numpy.logic.iscomplexobj

### PR DESCRIPTION
**Pull request by Sarvesh-Kesharwani**

### CODE REVIEW

File: ivy/functional/frontends/numpy/logic/truth_value_testing.py [hunk 1]
```diff
@@ -57,3 +57,22 @@ def isscalar(element):
 @to_ivy_arrays_and_back
 def isfortran(a: np.ndarray):
     return a.flags.fnc
+
```
Comment (ZoeCD) [inline ivy/functional/frontends/numpy/logic/truth_value_testing.py:60]: There are some link errors:
```
ivy/functional/frontends/numpy/logic/truth_value_testing.py:64:89: E501 line too long (92 > 88 characters)
ivy/functional/frontends/numpy/logic/truth_value_testing.py:78:1: W391 blank line at end of file
```
Did you installed the [pre-commit hook](https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit)?

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py [hunk 2]
```diff
@@ -141,3 +141,31 @@ def test_numpy_isfortran(
...
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=x[0],
+    )
+
+
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py:171]: Same here: `ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py:171:1: W391 blank line at end of file`

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py (hunk 2 above)
Comment (Sarvesh-Kesharwani) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py:171]: Hi ZoeCD i will install pre-commit and try to fix the lint errors,
thanks for highlighting.

**Comment by Sarvesh-Kesharwani** [comment]
Forwarded to #12377 PR

**Pull request closed without merging**